name: "Bug report üêõ"
description: Report errors or unexpected behavior
title: "[Bug]: "
labels: [Issue-Bug]
body:
- type: markdown
  attributes:
    value: |
      Please make sure to [search for existing issues](https://github.com/rebane2001/matterport-dl/issues)!
      Also check you can reproduce this with the latest version of matterport-dl. 
      Make sure your dependencies are all up to date (`pip install -U -r requirements.txt`) including your python version is at least the version mentioned in the readme: https://github.com/rebane2001/matterport-dl/blob/main/README.md
      Remember open source projects like this one are by volunteers the easier you make your bug to reproduce and limited variables the more likely/faster it will be resolved.

- type: input
  attributes:
    label: One or more sample matterport IDs / urls that reproduce the issue
    placeholder: "AAee22A"
    description: |
      IE: https://my.matterport.com/show/?m=3EjZgu2vnFc  or 3EjZgu2vnFc, 39awerJ291
      If this can only be reproduced with a private ID you can note that but it may be less likely to be resolved.      
  validations:
    required: true


- type: dropdown
  id: download
  attributes:
    label: Can you reproduce this issue with the sample small digital twin https://my.matterport.com/show/?m=EGxFGTFyC9N
    options:
      - I Don't Know, did not take the time to test it
      - 'Yes'
      - 'No'
      - Not applicable (ie bug the uses some matterport feature that you cannot use on the basic twin)
  validations:
    required: true

- type: input
  attributes:
    label: Started up message
    placeholder: "Started up a download run...."
    description: |
      If you are able to start matterport-dl.py it should say first thing a line like `Started up a download run Running python 3.2.4 on win32 with matterport-dl version: main (0fe3e39d81947e646ca3d4897c94c062020ae1af)`.
      Please include that line here if you have it.
  validations:
    required: false

- type: input
  attributes:
    label: Python version
    placeholder: "1.2.3"
    description: |
      If you were unable to put the startup message above, please put the python version here.  If you did include the startup message you can ignore this.
  validations:
    required: false

- type: textarea
  attributes:
    label: Steps to reproduce
    placeholder: Tell us the steps required to trigger your bug.
  validations:
    required: true

- type: textarea
  attributes:
    label: What went wrong
    placeholder: What happened instead? If the problem shows up when going to view the model look at the console output for any lines like `###### 404 error` these are likely new files from matterport we need to add to the downloader.
  validations:
    required: true


- type: textarea
  attributes:
    label: Workarounds
    description: If you already tried certain things or found a temporary/partial solution please let us know.  You can try `--no-advanced-download` to see if you can get even a partial download.
    placeholder: I tried X,Y,Z and had success with Y but Z caused applesauce.
  validations:
    required: false
